14/umbraco-cms/tutorials/multisite-setup.md

```diff
@@ -30,7 +30,7 @@ Keep in mind the [hostnames have to be configured in a specific way.](https://do
 
 ## Mapping the hostnames to individual websites/root nodes
 
-At this point, you should have several root nodes, where each is a separate website. To map the hostnames to individual root nodes:
+At this point, you should have several root nodes, where each is a separate website. By design, hostnames and cultures should be configured for each page, or you will experience 404 errors on the website. To map the hostnames to individual root nodes:
```

Comment:
Instead of adding a note here, I have another suggestion:

How about we add a step at the bottom of this list saying something like "Repeat steps 2-6" for each of your root nodes"?
What do you think?